packages/api/lib/testUtils.js

```diff
@@ -42,23 +43,27 @@ async function deleteAliases() {
   })));
 }
 
+function fakeFileFactory(params = {}) {
+  const name = randomId('name');
+
+  return {
+    bucket: randomString(),
+    name,
```

Comment:
Maybe a little bit out of scope, but are `name` and `filepath` always expected to have the same value? I'm wondering why we keep both around.

Reply:
They will not always be the same.  From the CNM docs:

> Some providers will only create hashes (to maximize read/write performance to/from S3 buckets). as object keys. We need a way to define what the object name should be, and explicitly told (not guessing).